src/rtc_base/platform_thread_types.cc

```diff
@@ -48,6 +52,8 @@ PlatformThreadId CurrentThreadId() {
   long tid;
   thr_self(&tid);
   return tid;
+#elif defined(WEBRTC_OPENBSD)
+  return static_cast<int64_t>(pthread_self());
```

Comment:
The right way is to define PlatformThreadId to the right type, not to add another variant of cast

Reply:
Right, I've added a proper definition and adjusted the cast accordingly.